tb_plugin/torch_tb_profiler/profiler/run_generator.py

```diff
@@ -332,7 +333,8 @@ def _generate_wait_graph(self):
 
     def _generate_ops_table(self):
         workers_to_comm_ops = OrderedDict()
-        for worker,data in self.all_profile_data.items():
+        # Ignore the span for distributed view
+        for (worker, span), data in self.all_profile_data.items():
```

Comment:
This need to be modified, because there may be data of same worker with different spans. We should aggregate data with same worker and multiple spans.